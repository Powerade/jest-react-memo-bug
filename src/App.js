import React from 'react';
import { MainMenu } from './MainMenu';

export default () => {
  return (
      <div id="app">
        <h1>My App</h1>
        <MainMenu />
      </div>
  );
};
