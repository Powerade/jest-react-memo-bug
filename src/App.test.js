import React from 'react';
import { shallow } from 'enzyme';
import App from './App';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import '@babel/polyfill';

Enzyme.configure({ adapter: new Adapter() });


describe('App', () => {
  test('should render My App', () => {
    expect(shallow(<App />)).toMatchSnapshot();
  });
});
