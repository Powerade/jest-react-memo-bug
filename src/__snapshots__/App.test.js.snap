// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render My App 1`] = `
<div
  id="app"
>
  <h1>
    My App
  </h1>
  <Component />
</div>
`;
